src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -43,7 +43,7 @@ defineProperties(ArcGisMapServerItemViewModel.prototype, {
      */
     type : {
         get : function() {
-            return 'esri-mapServer';
+            return 'esri-rest';
```

Comment:
Doesn't matter much to me.  arcgis/esri - rest/mapServer/mapservice seems to get used interchangeably.  data.gov.au has 2 formats and neither are named a combination of these.  Data.gov has Esri REST being most popular.  esri-mapService or esri-mapServer?